app/src/main/java/cm/aptoide/pt/view/app/Application.java

```diff
@@ -12,20 +12,20 @@
   private final long appId;
   private final String packageName;
   private final String tag;
-  private final boolean hasAppcIab;
-  private final boolean hasAppcAds;
+  private final boolean billing;
```

Comment:
I think that the name of the variable would make more sense if it had the verb prefix as it would help understanding the usage of the variable.